app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive: list[Animal] = []
+
+    def __init__(self, name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return f"{{Name: {self.name}," \
+               f" Health: {self.health}," \
+               f" Hidden: {self.hidden}}}"
```

Comment:
Try to avoid '\' use brackets instead
```suggestion
        return (
            f"{{Name: {self.name},"
            f" Health: {self.health}," 
            f" Hidden: {self.hidden}}}"
        )
```